app/data/Owners.scala

```diff
@@ -27,46 +27,55 @@ trait Owners {
 
 object Owners extends Owners {
 
-  override def default: Owner = Owner("dig.dev.tooling")
+  override def default: Owner = Owner("devx.sec.ops")
```

Comment:
I've replaced `dig.dev.tooling` with `devx.sec.ops` as the former also contains CSTI (and I don't think that they maintain many of the stacks that this ownership concept relates to)